mypy/fscache.py

```diff
@@ -46,31 +47,13 @@ def flush(self) -> None:
         self.listdir_error_cache = {}  # type: Dict[str, Exception]
```

Comment:
I have a local commit that replaces these to use cache_clear on the `lru_cache` decorated methods, but I didn't want to waste CI time by pushing it yet. I will do so when I have some good results from increasing the cache sizes.